.github/workflows/clojure.yml

```diff
@@ -0,0 +1,30 @@
+name: Run tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-clj:
+    strategy:
+      matrix:
+        # LTS and latest
+        jdk: [8,11,16]
```

Comment:
Note that .travis.yml says that openjdk6, openjdk7, and oraclejdk7 are also supported. While I'm not so sure about different JDK providers, I would at least consider supporting 6 and 7 still for this migration if possible.

Reply:
Looks like latest lein doesn't support jdk <8, and the travis build had been failing for quite some time because travis didn't support jdk <9. Let's just stick with LTS releases and the latest? That's also what e.g. metosin/malli does!

Reply:
I think this is a reasonable change. Seeming, for example, that TLS v1.3 has been backported to [JDK8](https://www.oracle.com/java/technologies/javase/8u261-relnotes.html), but not to JDK<8, it might make sense to stick with the LTS version as the minimum version anyway here. One arising question is, when do we stop supporting the LTS version once it has been deprecated? But okay, it doesn't need to be decided here, and we can make the decision to stop supporting 6 and 7—and if there is huge opposition from the crowd, we can always roll this back.

So, 8 and latest would be fine IMO.

Reply:
I think testing 11 since it's the latest LTS is a good idea, if we're testing multiple java versions anyway. At least I've used 11 a lot in production for stuff not in active development.

Reply:
Right, on top of latest, we probably want to test all the [LTS versions](https://www.oracle.com/java/technologies/java-se-support-roadmap.html), which is of course 8 and 11. 17 seems to also be an LTS, but it's not yet released so we address that one later (once setup-java's distributions start providing it).